app/assets/stylesheets/application.css

```diff
@@ -13,19 +13,72 @@
  *= require_tree .
  *= require_self
  */
+ .nav_links {
+   color: white;
+   padding: 14px 25px;
+   text-align: center;
+   text-decoration: none;
+   display: inline-block;
+   background-color: blue;
+ }
+
+
+ .shelter_name{
+   outline: none;
+   text-decoration: none;
+   display: inline-block;
+   margin-bottom: 1%;
+   width: 19.5%;
+   text-align: center;
+   line-height: 3;
+   color: black;
+ }
+ .shelter_name:link, .shelter_name:visited, .shelter_name:focus,
+ .delete_shelter:link, .delete_shelter:visited, .delete_shelter:focus,
+ .update_shelter:link, .update_shelter:visited, .update_shelter:focus
+  {
+   background: gray;
```

Comment:
This seems redundant to repeat all of these. I wonder if there is a way to make parent classes in html/css.